src/pps.mll

```diff
@@ -151,6 +170,19 @@ and gospel start_pos = parse
       scan lexbuf
     }
 
+and documentation start_pos = parse
+  | blank+ as s { Buffer.add_string buf s; documentation start_pos lexbuf }
+  | newline as nl { Buffer.add_string buf nl; Lexing.new_line lexbuf; documentation start_pos lexbuf }
+  | "*)" {
```

Comment:
This doesn’t support nested comments, but it would be nice to make sure they are, using a similar trick to was is done for regular comments.